pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -570,6 +575,47 @@ func GetVirtualCenterInstance(ctx context.Context,
 	return vCenterInstance, nil
 }
 
+// GetVirtualCenterInstanceForVCenterHost returns the vcenter object for given vCenter host
+// Takes in a boolean paramater reloadConfig.
+// If reinitialize is true, the vcenter object is instantiated again and the
+// old object becomes eligible for garbage collection.
+// If reinitialize is false and instance was already initialized, the previous
+// instance is returned.
+func GetVirtualCenterInstanceForVCenterHost(ctx context.Context,
+	vcconfig *VirtualCenterConfig, vcHost string, reinitialize bool) (*VirtualCenter, error) {
```

Comment:
May be we can skip vcHost as input param as we're already taking vcConfig as input which has has host string in it.

Reply:
done.